kasa/smartdevice.py

```diff
@@ -758,3 +746,42 @@ def __repr__(self):
             f" ({self.alias}), is_on: {self.is_on}"
             f" - dev specific: {self.state_information}>"
         )
+
+    @staticmethod
+    async def connect(
```

Comment:
`connect` vs. `create`, which sounds better?

Reply:
`connect` sounds better since its establishing and getting it ready to go